python/util.go

```diff
@@ -19,3 +22,14 @@ func runCmdStderr(cmd *exec.Cmd) error {
 	cmd.Stdout = os.Stderr
 	return cmd.Run()
 }
+
+func getVENVBinPath() (string, error) {
```

Comment:
Can you add a comment to getVENVBinPath clarifying that it returns the empty string for Docker?